adminSiteClient/PostsIndexPage.tsx

```diff
@@ -51,8 +70,36 @@ class PostRow extends React.Component<{
         this.saveTags(tags)
     }
 
+    @action.bound async onConvertGdoc() {
+        this.postGdocStatus = GdocStatus.CONVERTING
+        const { admin } = this.context
+        const json = await admin.requestJSON(
+            `/api/posts/${this.props.post.id}/createGdoc`,
+            {},
+            "POST"
+        )
+        this.postGdocStatus = GdocStatus.CONVERTED
+        this.props.post.gdocSuccessorId = json.googleDocsId
+    }
+
     render() {
         const { post, highlight, availableTags } = this.props
+        const { postGdocStatus } = this
+        const gdocElement = match(postGdocStatus)
+            .with(GdocStatus.MISSING, () => (
+                <button onClick={this.onConvertGdoc}>Create GDoc</button>
+            ))
+            .with(GdocStatus.CONVERTING, () => <span>Converting...</span>)
+            .with(GdocStatus.CONVERTED, () => (
+                <a
+                    href={`https://docs.google.com/document/d/${post.gdocSuccessorId}/edit`}
+                    target="_blank"
+                    rel="noopener"
+                >
+                    View GDoc
+                </a>
```

Comment:
We might want to use `<GdocsEditLink>` here, to make this action more recognizable across screens. It's also being added to the gdocs list view https://github.com/owid/owid-grapher/pull/1842